ush/oceanice_nc2grib2.sh

```diff
@@ -235,7 +235,9 @@ component=${1:?"Need a valid component; options: ice|ocean"}
 grid=${2:-"0p25"} # Default to 0.25-degree grid
 current_cycle=${3:-"2013100100"} # Default to 2013100100
 avg_period=${4:-"0-6"} # Default to 6-hourly average
-ocean_levels=${5:-"5:15:25:35:45:55:65:75:85:95:105:115:125"} # Default to 12-levels
+ocean_levels=${5:-"5:15:25:35:45:55:65:75:85:95:105:115\
```

Comment:
We don't use this script.  The global-workflow calls https://github.com/NOAA-EMC/global-workflow/blob/develop/scripts/exglobal_oceanice_products.py and https://github.com/NOAA-EMC/global-workflow/blob/develop/ush/python/pygfs/task/oceanice_products.py.  The levels are specified in the yaml https://github.com/NOAA-EMC/global-workflow/blob/develop/parm/post/oceanice_products.yam

Reply:
there is a copy of this in global-workflow https://github.com/NOAA-EMC/global-workflow/blob/ec634926f3b1ba1984b0735e17b76869659ceabe/ush/oceanice_nc2grib2.sh#L238

Reply:
Ok, Thanks for clarification!